docs/devGuide/development/workflow.md

```diff
@@ -126,6 +126,20 @@ The sections below has more information about various stages of submitting a PR.
 
   </modal>
 
+#### Keeping your fork up to date
+
+When new PRs are merged into the `master` branch, your `master` branch will be out of sync with the main repository. To update your branches to branch from the latest master:
+
+1. Go to your fork on Github and click 'Sync Fork'
+
+1. Checkout your `master` branch and `git pull`
+
+1. Checkout your feature branch and [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) the branch with `git rebase master` 
+
+1. Fix any merge conflicts (if applicable)
+
+You may want to use a tool like [GitTown](https://www.git-town.com/) to speed up the process.
+
```

Comment:
I think the docs is okay and we can add it.

Though I am thinking whether we should add this because this is not the only way.
I was thinking whether this will make them think this is the only right way.

Another possible way which I do is:
For me I add both remote repositories (upstream and origin(fork)) then I just sync them up via cli.

Reply:
Yea it is better!